src/routes/(app)/admin[[hash=admin_hash]]/nostr/+page.svelte

```diff
@@ -80,13 +82,19 @@
 </form>
 
 <h2 class="text-2xl">Relays</h2>
-
-<ul>
-	{#each data.nostrRelays as relay}
-		<li>{relay}</li>
-	{/each}
-</ul>
-
+<form action="?/addRelay" method="post" class="flex flex-col gap-4">
+	<ul>
+		{#each relays as relay, i}
+			<li>{relay}<button type="button" on:click={() => (index = i)}>🗑️</button></li>
```

Comment:
```suggestion
			<li>{relay}<button type="button" on:click={() => (relays = relays.filter(r => r !== relay))}>🗑️</button></li>
```

That way you can delete more than one relay at a time